src/services/find-available-port.service.js

```diff
@@ -12,13 +12,19 @@
  * hard of a problem!
  */
 const childProcess = window.require('child_process');
-
+const os = window.require('os');
 const MAX_ATTEMPTS = 15;
 
 export default () =>
   new Promise((resolve, reject) => {
     const checkPort = (port = 3000, attemptNum = 0) => {
-      childProcess.exec(`lsof -i :${port}`, (err, res) => {
+      // For Windows Support
+      // Similar command to lsof
+      // Finds if the specified port is in use
+      const command = /^win/.test(os.platform())
+        ? `netstat -aon | find "${port}"`
+        : `lsof -i :${port}`;
```

Comment:
Yay! Glad to see an equivalent exists (looks like it goes all the way back to Vista too, nice)